pkg/current/current.go

```diff
@@ -0,0 +1,38 @@
+package current
+
+import (
+	"fmt"
+	"os"
+	"path/filepath"
+	"regexp"
+
+	"github.com/tfversion/tfversion/pkg/download"
+	"github.com/tfversion/tfversion/pkg/helpers"
+	"github.com/tfversion/tfversion/pkg/use"
+)
+
+// CheckCurrentVersion prints the current active version of Terraform.
+func CheckCurrentVersion() error {
+
+	symlinkPath := filepath.Join(use.GetUseLocation(), download.TerraformBinaryName)
+	_, err := os.Lstat(symlinkPath)
+	if err != nil {
+		return fmt.Errorf("no active Terraform version found")
+	}
+
+	realPath, err := filepath.EvalSymlinks(symlinkPath)
```

Comment:
To find the number in a single line, you can do this:

```go
_, targetVersion := filepath.Split(realPath)
```

Check `alias.GetVersion()` for an existing implementation of this.

Reply:
Updated implementation in a9f5ec68ff3abaf7ca13c61b61f5e75f4ba61754

I had to call `filepath.Dir()` first, otherwise `filepath.Split()` would return me the binary name instead of the version number. Thank you for the suggestion, this method is a lot cleaner!